Modules/Core/Common/include/itkNeighborhood.hxx

```diff
@@ -70,12 +70,7 @@ template <typename TPixel, unsigned int VDimension, typename TContainer>
 void
 Neighborhood<TPixel, VDimension, TContainer>::SetRadius(const SizeValueType s)
 {
-  SizeType k;
-
-  for (DimensionValueType i = 0; i < VDimension; ++i)
-  {
-    k[i] = s;
-  }
+  SizeType k{ s };
```

Comment:
I don't think that's correct. I mean: `SizeType k{ s }` is equivalent to `SizeType k{ { s, 0, 0 } }` in 3-D.  If you want to "fill" all elements:

    auto k = SizeType::Filled(s);

Reply:
This is one of the mistakes that is making me step back and start anew.